test/clj/rems/test_browser.clj

```diff
@@ -549,6 +549,78 @@
                  last
                  (dissoc :event/id :event/time :event/attachments :event/actor-attributes)))))))
 
+(deftest test-invite-decider
+  (testing "create test data"
+    (btu/context-assoc! :form-id (test-helpers/create-form! {:form/fields [{:field/title {:en "description" :fi "kuvaus" :sv "rubrik"}
+                                                                            :field/optional false
+                                                                            :field/type :description}]}))
+    (btu/context-assoc! :catalogue-id (test-helpers/create-catalogue-item! {:form-id (btu/context-get :form-id)}))
+    (btu/context-assoc! :application-id (test-helpers/create-draft! "alice"
+                                                                    [(btu/context-get :catalogue-id)]
+                                                                    "test-invite-decider"))
+    (test-helpers/submit-application (btu/context-get :application-id) "alice"))
+  (btu/with-postmortem
+    (testing "handler invites reviewer"
+      (login-as "developer")
+      (go-to-application (btu/context-get :application-id))
+      (btu/wait-visible {:tag :h1 :fn/has-text "test-invite-decider"})
+
+      (btu/scroll-and-click :request-decision-dropdown)
+      (Thread/sleep 100) ;; need to wait for dropdown to expand
+      (btu/wait-visible :invite-decider-action-button)
+      (btu/scroll-and-click :invite-decider-action-button)
+
+      (btu/wait-visible :name-invite-decider)
+      (btu/fill-human :name-invite-decider "Malice Applicant")
+      (btu/fill-human :email-invite-decider "malice@example.com")
+      (btu/scroll-and-click :invite-decider)
+      (btu/wait-visible {:css ".alert-success"})
+      (logout))
+    (testing "get invite token"
+      (let [[token invitation] (-> (btu/context-get :application-id)
+                                   applications/get-application-internal
+                                   :application/invitation-tokens
+                                   first)]
+        (is (string? token))
+        (is (= {:application/decider {:name "Malice Applicant" :email "malice@example.com"}
```

Comment:
Why do you want to use malice and carl here? Is there some significance in using completely different values? Also could create a new user too.

Reply:
Creating a fresh user makes sense, I'll do that